### PR TITLE
Bump OpenIddict to 3.0.0-rc1.20567.82

### PR DESCRIPTION
**Pull request by kevinchalet**
Fixes https://github.com/OrchardCMS/OrchardCore/issues/7629.

https://kevinchalet.com/2020/11/17/introducing-openiddict-3-0-s-first-release-candidate-version/

### CODE REVIEW

**Comment by hishamco** [comment]
Build is failed

**Comment by kevinchalet** [comment]
I know. That's because it requires the .NET 5 SDK, that OC doesn't use it yet. We're discussing this change on Gitter ATM.

**Comment by Skrypt** [comment]
Should it not target then https://github.com/OrchardCMS/OrchardCore/tree/ag/net5 instead?

**Comment by kevinchalet** [comment]
@Skrypt no: according to @agriffard, this branch - that makes a full migration to .NET 5.0 (SDK + TFMs + packages dependencies) will only be merged after OC 1.0 ships.

In this case, I don't need OC to target .NET 5.0. I just need the 5.0 SDK to fix a bug that affects nullable references.

File: src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs [hunk 1]
```diff
@@ -99,7 +97,7 @@ public async Task<IActionResult> Authorize()
                     {
                         [OpenIddictServerAspNetCoreConstants.Properties.Error] = Errors.ConsentRequired,
                         [OpenIddictServerAspNetCoreConstants.Properties.ErrorDescription] =
-                            S["The logged in user is not allowed to access this client application."]
+                            "The logged in user is not allowed to access this client application."
```
Comment (agriffard) [inline src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs:100]: No more `S[]` ?

Comment (kevinchalet) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs:100]: It's deliberate. See https://github.com/OrchardCMS/OrchardCore/pull/7675/files#diff-44ca9283b8f821a762e5d0c72ad06292720058093eeb604fb872af0aee4f77a3R26-R27 for the explanation.

**Comment by kevinchalet** [comment]
I ended up tweaking the `.yml` files to install both the .NET 5.0 and .NET Core 3.1 SDKs. Technically, we don't need the 3.1 SDK, but we need the 3.1 runtime to run the tests, and installing the entire SDK is the likely simplest option.

If OC used Arcade, it would take care of downloading the runtimes automatically. That's what we use in OpenIddict: https://github.com/openiddict/openiddict-core/blob/dev/global.json#L2-L11

**Review by sebastienros — APPROVED** [review]

**Pull request merged**